packages/ui-react/src/components/TextField/TextField.tsx

```diff
@@ -15,6 +15,12 @@ type InputOrTextAreaProps =
   | ({ multiline?: never; inputRef?: RefObject<HTMLInputElement>; textAreaRef?: never } & InputProps)
   | ({ multiline: true; inputRef?: never; textAreaRef?: RefObject<HTMLTextAreaElement> } & TextAreaProps);
 
+type ConditionalWrapperProps = {
```

Comment:
@langemike I like the approach of keeping this type reusable by integrating it with the TextField types.
  Would this be better suited to live in the `/packages/ui-react/types` ?

What are the thoughts on creating react utility types for reusability

Reply:
Good idea, I will add it. Was thinking about something similar when I originally wrote it

Reply:
I approached the conditional rendering of HTML differently, because I noticed an unwanted side effect on Android while testing.

When you use the onscreen keyboard, when typing in a input field, the keyboard will hide on every keypress. 😕

I think it has something to do with the dynamically updated `value=""` attribute and React internally decides it needs a re-render or something.

With this new approach it doesn't happen anymore.